src/controller/python/chip/setup_payload/setup_payload.py

```diff
@@ -95,3 +95,10 @@ def __InitNativeFunctions(self, chipLib):
         setter.Set("pychip_SetupPayload_ParseManualPairingCode",
                    c_int32,
                    [c_char_p, SetupPayload.AttributeVisitor, SetupPayload.VendorAttributeVisitor])
+
```

Comment:
Should we add a demarcator  here to 'contain' our changes so we can work only in this area in the future, like this or 🤷 : 
######--------------------------------------------------######

Reply:
Sure! Keep our change ours.